core/tests/webdriverio_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -151,15 +152,18 @@ var ExplorationEditorFeedbackTab = function() {
     await action.setValue(
       'Feedback Response Text Area',
       feedbackResponseTextArea, feedbackResponse);
-    await action.click('Feedback Status Dropdow', feedbackStatusDropdown);
-    var optionLabelFeedbackStatus = (
-      $(`option[label="${feedbackStatus}"]`));
-    await action.click(
-      'Option[label = "feedback status"', optionLabelFeedbackStatus);
+    await waitFor.presenceOf(
+      feedbackStatusDropdown, 'Feedback Status Dropdown is not visible');
```

Comment:
Replaced with visibiltyOf. Also we need to wait for visibility of because previously we were doing the same while using action.click for selecting the option.